ci/test.sh

```diff
@@ -26,13 +31,17 @@ function main {
   [[ $# -eq 0 || "$1" == '-h' || "$1" == '--help' ]] && helpAndExit 0
```

Comment:
please check if there exist 2 parameters.

Reply:
We are not checking the number of parameter in any of the other files. Do you still want to check the number of parameters for all the scripts files?

Reply:
Just checked your point. True that. Not necessary